src/core/toga/interface/widgets/base.py

```diff
@@ -331,3 +330,21 @@ def set_font(self, font):
         :type  font: :class:`toga.Font`
         """
         self._set_font(font)
+
+    def hide(self, widget):
```

Comment:
The idea is to use like this : self.outer_box.hide(self.label_box), but yeah I can update to self.label_box.hide().